livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -679,6 +683,9 @@ ros::Publisher *Lddc::GetCurrentPublisher(uint8_t handle) {
           name_str, queue_size);
     }
   }
+  else {
+    std::runtime_error("Couldn't find a suitable publisher for Livox Lidar");

```

Comment:
I don't think that this is correct... it looks like the logic is ensuring that a publisher exists, even if one hasn't been created before.

Reply:
What do you propose? Reverting the change?

Reply:
if publisher doesn't exist:
    create publisher
    set pub pointer
else
    // Do nothing.
    
return pointer to publisher.

Reply:
Ok, reverted.